### PR TITLE
convert kafka pillow to use sql checkpoints

### PR DESCRIPTION
**Pull request by czue**
depends on https://github.com/dimagi/pillowtop/pull/172

wait for tests

### CODE REVIEW

**Comment by czue** [comment]
cc @snopoke @calellowitz

File: corehq/apps/change_feed/consumer/pillow.py [hunk 1]
```diff
@@ -2,18 +2,21 @@
...
 # in experimental stage
-KAFKA_CHECKPOINT_FREQUENCY = 10000
+KAFKA_CHECKPOINT_FREQUENCY = 1000
 
 
 def get_demo_case_consumer_pillow():
-    document_store = CouchDocumentStore(CommCareCase.get_db())
+    document_store = DjangoDocumentStore(
+        DjangoPillowCheckpoint, DjangoPillowCheckpoint.to_dict, DjangoPillowCheckpoint.from_dict,
+    )
```
Comment (snopoke) [inline corehq/apps/change_feed/consumer/pillow.py]: Instantiating the doc store this way means I can only be used for checkpoints right? Maybe call it `checkpoint_doc_store` or some such things to make it more clear.

Comment (czue) [inline corehq/apps/change_feed/consumer/pillow.py]: :+1:

**Comment by czue** [comment]
fyi, i had a "don't merge" on this because i wanted to write a migration for the couch checkpoint and test it. no biggie though, i can test on the next one.

**Comment by czue** [comment]
(also i added it after I wrote the original description so could see how that was misleading)

**Comment by snopoke** [comment]
Sorry, I did see that and then later forgot

On 6 November 2015 at 08:32, Cory Zue <notifications@github.com> wrote:

> (also i added it after I wrote the original description so could see how
> that was misleading)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8957#issuecomment-154324023>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

**Pull request merged**